core/common/assets/js/common.js

```diff
@@ -42,8 +42,7 @@ class ElementorCommonApp extends elementorModules.ViewModule {
 
 		this.api = window.$e;
 
-		// TODO: UNCOMMENT
-		//$e.components.register( new EventsDispatcherComponent() );
+		$e.components.register( new EventsDispatcherComponent() );
```

Comment:
Because event tracking is not only going to be in the onboarding. It is in the common because it will be utilized across the entire site (admin, editor, admin-frontend) @matipojo